src/Service/CprHelper.php

```diff
@@ -205,4 +205,12 @@ public function collectRelevantData(array $data): array
 
         return $relevantData;
     }
+
+    /**
+     * Removes potential dashes from CPR.
+     */
+    public function formatIdentifier(string $cpr): string
+    {
+        return str_replace('-', '', $cpr);
```

Comment:
I would use `preg_replace('/[^0-9]+/', '', $cpr)` to make sure that only digits are kept. The current solution works given all the other checks, but you'll be surprised to learn some of the weird ways people can find to write “a number”, e.g. `123456 7890` or ` 123456/7890`.

When programming and filtering stuff, it's always a good idea to only keep what you want rather than throwing away what you don't want because it often easier to know what you want than what you don't want.

Reply:
Used `preg_replace('/[^0-9]+/', '', $cpr)`.